client/src/main.ts

```diff
@@ -86,7 +86,9 @@ async function startVue () {
       ],
       ignoreErrors: [
         // this error is safe to ignore, ref: https://stackoverflow.com/a/50387233/8679162
-        'ResizeObserver loop limit exceeded'
+        'ResizeObserver loop limit exceeded',
+        // ignore Launch Darkly errors (partial match)
+        'LD: [error]'
```

Comment:
Added another partial string match (regex is also supported) to ignore LD errors.